openfl/net/URLRequest.hx

```diff
@@ -13,7 +13,10 @@ import openfl.utils.ByteArray;
 	public var requestHeaders:Array<URLRequestHeader>;
 	public var url:String;
 	public var userAgent:String;
-	
+
+	#if (js && html5)
```

Comment:
Thats the way most of the openfl sources do it, so I did it the same.

And I think int is not the same, since `js` (node) in not `html5` (browser)